docker-compose.production.yml

```diff
@@ -0,0 +1,63 @@
+version: "3.2"
```

Comment:
let's get these dockerfiles all on teh same version. also, yaml files should always start with
```
---
```

Reply:
Done!